cluster/olm/ceph/assemble/metadata-common.yaml

```diff
@@ -21,6 +21,11 @@ spec:
         version: v1
         displayName: Ceph Object Store User
         description: Represents a Ceph Object Store User.
+      - kind: CephNFS
+        name: cephnfses.ceph.rook.io
+        version: v1
+        displayName: Ceph NFS
+        description: Represents a cluster of Ceph NFS ganesha gateways.
   displayName: Rook-Ceph
```

Comment:
Please add:

`replaces: rook-ceph.v1.0.1`

Reply:
done

Reply:
moved the release stuff to another PR